app/src/test/java/org/mozilla/fenix/share/ShareControllerTest.kt

```diff
@@ -108,6 +108,8 @@ class ShareControllerTest {
         testController.handleShareToApp(appShareOption)
         advanceUntilIdle()
 
+        assertEquals(appPackageName, Events.shareToApp.testGetValue()?.last()?.extra?.getValue("app_package"))
```

Comment:
Could we also add a null assertion before `handleShareToApp`

Reply:
Done